src/govuk/components/radios/radios.yaml

```diff
@@ -380,6 +383,46 @@ examples:
           <label class="govuk-label" for="contact-text-message">Mobile phone number</label>
           <input class="govuk-input govuk-!-width-one-third" name="contact-text-message" type="text" id="contact-text-message">
 
+- name: prechecked
```

Comment:
Is it worth adding an example just for demonstrating the `value` option? You've added a couple here but my read of this is that they're for demonstrating different things alongside the `value` option over demonstrating the.. er... value of the `value` option on its own as you have done in the checkboxes example.